Cookbook/Technical-Documents/WritingAProposal.md

```diff
@@ -0,0 +1,37 @@
+Writing a new proposal
+======================
+
+## When to write proposals
+
+When we want to make a significant change to our process, architecture, codestyle or techniques, we require to go through the Proposal process.
+
+## How to write a new proposal
+
+### Write a proposal document
+
+* Create a copy of [the proposal template](../Proposals/Template_Proposal.md)
+* Fill all the fields and paragraphs, describing your proposal the best you can.
+* Be sure to be clear in your explanation of the problem to solve, the proposed solutions, but also potential limitations and the alternatives considered
+
+### Make a Pull Request
+
+* Describe in a few words the purpose of the proposal in the PR body. Also mention any previous discussion (e.g. Slack) on the topic.
+* Copy-paste [the template dedicated to proposal PRs](https://raw.githubusercontent.com/Babylonpartners/ios-playbook/master/.github/PULL_REQUEST_TEMPLATE/proposal.md) at the end and fill it
```

Comment:
or add a `template=proposal.md` query parameter to the URL when you are creating a pull request so that it is pasted for you in the pull request body automatically

Reply:
@ilyapuchka That was my original idea too, but to create the URL you need to also know the name of the branch 😢 

`https://github.com/Babylonparners/ios-playbook/compare/master...NAMEOFBRANCH?quick_pull=1&template=proposal.md`